.changeset/thick-panthers-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+"@manypkg/cli": minor
+---
+
+Replace fs-extra with native node:fs/promises
```

Comment:
```suggestion
Replace `fs-extra` with native `node:fs/promises`
```

Reply:
@Andarist ill update the other PRs with this formatting too, so don't bother changing it there.